public/src/test/java/com/rackspace/salus/telemetry/api/web/client/TestMonitorControllerTest.java

```diff
@@ -0,0 +1,136 @@
+ * limitations under the License.
+ */
+
+package com.rackspace.salus.telemetry.api.web.client;
+
+import static com.rackspace.salus.common.util.SpringResourceUtils.readContent;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rackspace.salus.event.manage.model.TestTaskResult;
+import com.rackspace.salus.event.manage.model.TestTaskResult.EventResult;
+import com.rackspace.salus.event.manage.model.kapacitor.KapacitorEvent.EventData;
+import com.rackspace.salus.event.manage.model.kapacitor.KapacitorEvent.SeriesItem;
+import com.rackspace.salus.event.manage.model.kapacitor.Task.Stats;
+import com.rackspace.salus.telemetry.api.config.ApiPublicProperties;
+import com.rackspace.salus.telemetry.api.config.ServicesProperties;
+import com.rackspace.salus.telemetry.api.model.TestMonitorAndEventTaskRequest;
+import com.rackspace.salus.telemetry.api.model.TestMonitorAndEventTaskResponse;
+import com.rackspace.salus.telemetry.api.services.TestMonitorAndEventTaskService;
+import com.rackspace.salus.telemetry.api.web.TestMonitorController;
+import com.rackspace.salus.telemetry.api.web.TraceResponseFilter;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import uk.co.jemos.podam.api.PodamFactory;
+import uk.co.jemos.podam.api.PodamFactoryImpl;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(controllers = TestMonitorController.class)
+public class TestMonitorControllerTest {
+
+  @Autowired
+  MockMvc mockMvc;
+
+  @Autowired
+  ObjectMapper objectMapper;
+
+  private PodamFactory podamFactory = new PodamFactoryImpl();
+
+  @Autowired
+  TestMonitorController testMonitorController;
+
+  @MockBean
+  TraceResponseFilter traceResponseFilter;
+
+  @MockBean
+  ApiPublicProperties apiPublicProperties;
+
+  @MockBean
+  TestMonitorAndEventTaskService testMonitorAndEventTaskService;
+
+  @MockBean
+  ServicesProperties servicesProperties;
+
+  @Test
+  public void testCreateTestMonitorAndEventTask_Success() throws Exception {
+    String tenantId = RandomStringUtils.randomAlphabetic(8);
+    TestMonitorAndEventTaskRequest testMonitorAndEventTaskRequest = objectMapper
+        .readValue(readContent("testCreateTestMonitorAndEventTask_req.json"),
```

Comment:
```suggestion
        .readValue(readContent("testPerformTestMonitorAndEventTask_req.json"),
```

Changing these everywhere would be good.